synapse/logging/opentracing.py

```diff
@@ -1032,11 +1034,11 @@ def trace_servlet(
             # with JsonResource).
             scope.span.set_operation_name(request.request_metrics.name)
 
-            # set the tags *after* the servlet completes, in case it decided to
-            # prioritise the span (tags will get dropped on unprioritised spans)
```

Comment:
Moved the comment because it seems more relevant where we actually set the tags, not fil it out more.